### ISSUE TITLE
Negative sample loss function

### ISSUE THREAD

**Issue by cp717** [tjiiv-cprg/SPFCN-ParkingSlotDetection#12]
请问torch.pow(1 - mark_gt[neg_gt], 4)是什么意思？还有为什么不对mark_neg_loss做归一化                  
mark_neg_loss = torch.pow(mark_neg, 2) * torch.log(1 - mark_neg) * torch.pow(1 - mark_gt[neg_gt], 4)
